config/sync/core.entity_view_display.paragraph.simple_links.default.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 5125187b-1e73-4d32-84a5-ed8f3e5dfe11
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.simple_links.field_link
+    - paragraphs.paragraphs_type.simple_links
+  module:
+    - link
+id: paragraph.simple_links.default
+targetEntityType: paragraph
+bundle: simple_links
+mode: default
+content:
+  field_link:
+    type: link
```

Comment:
Should this be `linkit`?

Reply:
Good catch, but this is the `entity_view_display`and not the `entity_form_display`. In our case i think we only to ensure the field is display'ed as linkit to the editors.